cherrypy/__init__.py

```diff
@@ -57,9 +57,14 @@
 """
 
 try:
-    import pkg_resources
+    import importlib.metadata as importlib_metadata
 except ImportError:
-    pass
+    # fall back for python <= 3.7
+    # can simply pass when py 3.6/3.7 no longer supported
```

Comment:
Actually, not even a try-except would be needed under 3.8+

@radez I just realized that we also forgot to add `importlib_metadata` to `install_requires` with a conditional env marker.

Reply:
Oh yea! I totally forgot about that too!